data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,103 @@ mission "Arachi Cartoonist 2"
+			choice
+				`	(Help keep the main gate closed.)`
+					goto defww
+				`	(Go with the others check the back gate.)`
+			`	Once you get to the back gate, you and the Kimek there stay quiet, to try and listen for anyone approaching it outside of the fort. Sure enough, you hear many taps on the grass, as the attackers come circling around. Fortunately, not many came to this gate, so you and the Kimek don't struggle much to keep it closed. The group at the front gate isn't so lucky, however, and after a brief minute the whistle sounds again, and your group comes back to the courtyard to find that the gate has been busted open, and that the attackers have won.`
+				goto lost
+			label defww
+			`	At first, holding the gate isn't too troublesome, but after a few minutes pass, it starts being pounded by some form of battering ram, and before long the group there can't hold it closed. At the same time, you hear a commotion from behind the shack, and as you and others go there to check, you arrive to see the small gate open. The attackers having made their way inside via both gates, the whistle sounds again, marking the end of the battle, and the victory of the attackers.`
+				goto lost
+			label att
+			`	He brings you to the nearby camp, where many Kimek, Arachi, and even a few Saryds are gathered for the event, waiting in anticipation inside one of the tents, or looking up at the fort. All of them have long, blue strips of cloth over their own clothing. Once she notices you two, another Kimek organizer, donning what looks like a thin gambeson, comes greet you two. She greets her colleague, who introduces you and tells her you chose to join the attackers, before leaving the camp to welcome a family who has come to watch the event.`
+			`	"Honored to have you fight alongside us, our company is," the armored Kimek begins. "Told about our objective, I assume you have been. Break into the fort, we must, before up the time is. Ten minutes, we have, once started, the battle has." She tells you to follow here to the 'war tent,' one that's slightly larger than the other ones around, and gives you a long piece of blue cloth, telling you to wrap it around you, to mark what team you are on. Wrapping it around like a scarf, you look around the main tent to see that stored there is a stumpy log, reinforced with metal bits, which the Kimek tells you will be used as a battering ram to force open the gates. She then has you follow her to the edge of the camp, showing you the limits of this "set," marked by the patches of vegetation, telling you that you're not allowed to go past them to not disrupt other sets. "Also, against the rules, it is, to crawl up the walls of the fort, so," she pauses, looking at you as if realizing you can't crawl up walls. "Sorry, out of habit, I said it. Participate in this event, not many groundlocked people do, few Saryds here I've met. Even fewer humans, as imagine, you can."`
```

Comment:
In the Coalition Games PR, most of what the Kimek do hinges on them being able of crawling up walls, so I'm not of mind to remove that ability from them.

Reply:
Ah, then this does present something of an issue, because walls really make no sense as a defence emplacement if that is the case.

Reply:
That is pretty much acnowledged by the Kimek attacker leader, and if you ask the Kimek that offers you the mission about it being reenacted, he'll spell it out that it's not something meant to make perfect sense.
Such is the case for the Saryd stories, too. A moving tree saves a boy from a storm that appears to be hunting him? A whale flies up to the skies to save a fisherman from the waves? A huntress doesn't realize a gigantic beast is right behind her, and is saved by the bird she was hunting for some reason?
These folklore missions aren't meant to give detailed historical or  archeological information on any of the species, just provide an outlet to share some of their culture.

Reply:
Of course not, no I understand that, but there are certain underlying principles to folklore. 

Moving trees, flying fish, talking spirits, visions in water, etc. take your pick of possible things, the fact that they're handwavy myth elements is not an issue. They exist to account for the metaphor of a story that might otherwise not be very interesting, or are simply things added to account for gaps left in tall tales told by previous generations.

Some constants remain though. A house in a folk tale still has walls and a roof, even if the house grows legs and walks around. A road may be magical, made of yellow bricks, and lead to wherever it needs to go, but it is still a paved path from one place to another.

Defensive emplacements exist to act as force multipliers, they work for us because we're bad at climbing flat surfaces. Whereas if Kimek can climb them fairly easily, then the simple _concept_ of using a wall defensively would never have entered the society in the first place. So it wouldn't have ended up in folklore.

However, I have a thought that might help.
Humans migrated to locations we weren't well suited for on earth very early in our history, and we developed clothing way back even before fire. Technological adaptations to overcome our physical limitations are many. Perhaps the Kimek are indeed bad at climbing but developed some sort of foot wear that allowed them to climb very effectively at the same time that they started building cities to be taller, in much the same way that we developed bicycles to get around more quickly when our cities started growing to be bigger than one could reasonably walk across?

This way your mythos folk tale could simply be the equivalent of a pre-industrial fairytale, from before they developed the Kimek equivalent of suction shoes or something?

Reply:
Perhaps Kimek are only able to climb on some surfaces and not others, and the defensive walls are made of such a substance?

Reply:
Perhaps, but for it to work as a folk-lore type thing that material would have to be; natural (wood, stone, etc.), workable with rudimentary tools (like, hack it with a rock and tie it together with string or gravity), and abundant on their world.

Very smooth things like glass or plastic are the obvious surfaces for not being climbable, but for the concept of using a wall defensively to enter into the pre-industrial consciousness it needs to be able to be made of something you'd have in at least moderate abundance with pre-industrial technologies. I don't have a good answer for what that might be.

Reply:
> Ah, then this does present something of an issue, because walls really make no sense as a defence emplacement if that is the case.

I agree with walls being less useful, but they still offer some advantages that could probably justify their usage. Like blocking sight and offering a better overview for the defenders, both valuable tactical assets. Why the attackers would prefer to enter through gates might still be a problem, but I guess that could be explained by the folklore nature and the fact that every species is allowed to participate.

Reply:
You can block sight with a moderate mound of dirt, which is dramatically easier to build, won't be destroyed or damaged, and can still provide much of the same defensive advantage in melee. A tower provides a comparatively superior overview, and would be vastly more useful than a wall for that purpose. Especially since the Kimek aren't exactly equipped physically to use low-tech ranged weapons so being exposed on a tower isn't such a hazard. (Also, a safe place to gain advantage using ranged weapons is a big driver for the existence of better walls, which again has no bearing on the Kimek.)

Like, constraining the method of attack for the game to fit the story is fine. There's still fundamentally nothing that a wall does that isn't done better and easier by something else if you're capable of readily climbing it - which means the mere concept of using an actual high wall _as a defence_ simply doesn't make sense. It would've never entered their species' minds until they encountered alien cultures after achieving spaceflight, which is too late for it to be folklore.

Reply:
Actually I just remembered I was supposed to have added a part saying how the outside of the walls also has wooden spikes all over, not the type that could actually hurt somebody in this case but just to make climbing them not worth it.
Would that be alright?

Reply:
Rather than those spikes being manually added (which is an intensive construction effort and difficult to maintain as they weather), could it instead be that the tree they use for construction naturally grows downward slanted thorns that prevent climbing? (Like big thorns, half a metre or more in length. Or vicious thickety thorns like these:)
![image](https://user-images.githubusercontent.com/60949828/115641152-ac1d0f80-a357-11eb-8bc7-db5102af56f4.png)

This just occurred to me, and if all you have to do is take one out and substitute a new one in when/if the thorns get damaged beyond use, then as a natural, renewable, and classically available resource this could justify how they came to this development.

Reply:
Didn't know something like that even existed.
Sure, can be those.